react-frontend/src/components/Home/infocard.js

```diff
@@ -20,12 +20,12 @@ function InfoCard({
     isRouteCard = true;
   }
 
-  cardHeight = height == null ? '300px' : height;
+  cardHeight = height == null ? '600px' : height;
```

Comment:
null is falsy so the simpler way of coding this is 
cardHeight = height ? height :  '600px' ;